### DISCUSSION TITLE
rioxarray projection error NotGeoreferencedWarning

### DISCUSSION THREAD

**Discussion by billmaster1** [corteva/rioxarray#631] [Q&A]
I'm using rioxarray to clip some .nc files processed through Polymer (atmospheric processor). The issue I'm having is when I import the .nc file I get the following error:

The original file was in projection UTM18/EPSG32618

NotGeoreferencedWarning: Dataset has no geotransform, gcps, or rpcs. The identity matrix will be returned.

I've dumped the .nc file info and it looks like the x,y dimensions are in meters. I'm not that familiar wiht netcdf file formats ... wondering if there is a way to set the projection. I don't see any projection info contained in the ncdump file attached.

Any help anyone can provide would be greatly appreciated.

Thanks

output from ncdump ...

netcdf test {
dimensions:
    width = 10980 ;
    height = 10980 ;
variables:
    double latitude(height, width) ;
        latitude:_FillValue = 9.96920996838687e+36 ;
    double longitude(height, width) ;
        longitude:_FillValue = 9.96920996838687e+36 ;
    short bitmask(height, width) ;
        bitmask:_FillValue = -32767s ;
        bitmask:description = "LAND:1, CLOUD_BASE:2, L1_INVALID:4, NEGATIVE_BB:8, OUT_OF_BOUNDS:16, EXCEPTION:32, THICK_AEROSOL:64, HIGH_AIR_MASS:128, EXTERNAL_MASK:512, CASE2:1024, INCONSISTENCY:2048, ANOMALY_RWMOD_BLUE:4096" ;
        bitmask:bitmask_reject = "bitmask & 1023 != 0" ;
    float Rnir(height, width) ;
        Rnir:_FillValue = 9.96921e+36f ;
    float Rgli(height, width) ;
        Rgli:_FillValue = 9.96921e+36f ;
    float logchl(height, width) ;
        logchl:_FillValue = 9.96921e+36f ;
        logchl:description = "log10 of the chl-a concentration in mg/m3" ;
    float bbs(height, width) ;
        bbs:_FillValue = 9.96921e+36f ;
    float Rw443(height, width) ;
        Rw443:_FillValue = 9.96921e+36f ;
        Rw443:description = "water reflectance (dimensionless ; fully normalized)" ;
    float Rw490(height, width) ;
        Rw490:_FillValue = 9.96921e+36f ;
        Rw490:description = "water reflectance (dimensionless ; fully normalized)" ;
    float Rw560(height, width) ;
        Rw560:_FillValue = 9.96921e+36f ;
        Rw560:description = "water reflectance (dimensionless ; fully normalized)" ;
    float Rw665(height, width) ;
        Rw665:_FillValue = 9.96921e+36f ;
        Rw665:description = "water reflectance (dimensionless ; fully normalized)" ;
    float Rw705(height, width) ;
        Rw705:_FillValue = 9.96921e+36f ;
        Rw705:description = "water reflectance (dimensionless ; fully normalized)" ;
    float Rw740(height, width) ;
        Rw740:_FillValue = 9.96921e+36f ;
        Rw740:description = "water reflectance (dimensionless ; fully normalized)" ;
    float Rw783(height, width) ;
        Rw783:_FillValue = 9.96921e+36f ;
        Rw783:description = "water reflectance (dimensionless ; fully normalized)" ;
    float Rw842(height, width) ;
        Rw842:_FillValue = 9.96921e+36f ;
        Rw842:description = "water reflectance (dimensionless ; fully normalized)" ;
    float Rw865(height, width) ;
        Rw865:_FillValue = 9.96921e+36f ;
        Rw865:description = "water reflectance (dimensionless ; fully normalized)" ;
    float Rw1610(height, width) ;
        Rw1610:_FillValue = 9.96921e+36f ;
        Rw1610:description = "water reflectance (dimensionless ; fully normalized)" ;

// global attributes:
        :dir_base = "/work/ESTUARY/sentinel/tools/polymer/polymer-v4.14" ;
        :sensor = "MSI" ;
        :dir_common = "/work/ESTUARY/sentinel/tools/polymer/polymer-v4.14/auxdata/common/" ;
        :thres_Rcloud = "0.2" ;
        :thres_Rcloud_std = "0.04" ;
        :force_initialization = "False" ;
        :reinit_rw_neg = "False" ;
        :max_iter = "100" ;
        :size_end_iter = "0.005" ;
        :metrics = "W_dR2_norm" ;
        :glint_precorrection = "True" ;
        :external_mask = "None" ;
        :lut_file = "/work/ESTUARY/sentinel/tools/polymer/polymer-v4.14/auxdata/generic/LUT.hdf" ;
        :thres_chi2 = "0.005" ;
        :partial = "0" ;
        :weights_corr = "None" ;
        :weights_oc = "None" ;
        :atm_model = "T0,-1,Rmol" ;
        :normalize = "3" ;
        :Rprime_consistency = "True" ;
        :water_model = "PR05" ;
        :initial_point_1 = "[-1, 0]" ;
        :initial_point_2 = "[1, 1]" ;
        :initial_step = "[0.2, 0.2]" ;
        :bounds = "[[-2, 2], [-3, 3]]" ;
        :constraint_bbs = "[0.0001, 0.2258, 0.9233]" ;
        :bbopt = "0" ;
        :min_abs = "0" ;
        :absorption = "bricaud98_aphy" ;
        :no2_climatology = "/work/ESTUARY/sentinel/tools/polymer/polymer-v4.14/auxdata/common/no2_climatology.hdf" ;
        :no2_frac200m = "/work/ESTUARY/sentinel/tools/polymer/polymer-v4.14/auxdata/common/trop_f_no2_200m.hdf" ;
        :multiprocessing = "-1" ;
        :altitude = "0" ;
        :BITMASK_INVALID = "551" ;
        :BITMASK_REJECT = "1023" ;
        :verbose = "True" ;
        :dbg_pt = "[-1, -1]" ;
        :bands_corr = "[443, 490, 560, 665, 705, 740, 783, 865]" ;
        :bands_oc = "[443, 490, 560, 665, 705, 740, 783, 865]" ;
        :bands_rw = "[443, 490, 560, 665, 705, 740, 783, 842, 865, 1610]" ;
        :band_cloudmask = "865" ;
        :calib = "{443: 1.0, 490: 1.0, 560: 1.0, 665: 1.0, 705: 1.0, 740: 1.0, 783: 1.0, 842: 1.0, 865: 1.0, 945: 1.0, 1375: 1.0, 1610: 1.0, 2190: 1.0}" ;
        :K_OZ = "{443: 0.002897, 490: 0.02066, 560: 0.1058, 665: 0.05005, 705: 0.01887, 740: 0.008743, 783: 0.0077, 842: 0.0, 865: 0.002156, 945: 0.0, 1375: 0.0, 1610: 0.0, 2190: 0.0}" ;
        :K_NO2 = "{443: 0.0, 490: 0.0, 560: 0.0, 665: 0.0, 705: 0.0, 740: 0.0, 783: 0.0, 842: 0.0, 865: 0.0, 945: 0.0, 1375: 0.0, 1610: 0.0, 2190: 0.0}" ;
        :Ncoef = "3" ;
        :processing_duration = "0:21:09.051298" ;
        :l1_filename = "/work/ESTUARY/sentinel/data_downloads/run_4594619/safe/S2A_MSIL1C_20170903T160131_N0205_R054_T18SVJ_20170903T160146.SAFE/GRANULE/L1C_T18SVJ_A011491_20170903T160146" ;
        :sensing_time = "2017-09-03 16:01:46" ;
        :L1_TILE_ID = "S2A_OPER_MSI_L1C_TL_SGS__20170903T210554_A011491_T18SVJ_N02.05" ;
        :L1_DATASTRIP_ID = "S2A_OPER_MSI_L1C_DS_SGS__20170903T210554_S20170903T160146_N02.05" ;
        :central_wavelength = "{443: 442.6950461040809, 490: 492.4365766613168, 560: 559.849055488363, 665: 664.6217529192422, 705: 704.1149362235096, 740: 740.4918208862697, 783: 782.7529173400575, 842: 832.7904111482434, 865: 864.710789243533, 945: 945.0544704403069, 1375: 1373.461882545774, 1610: 1613.659406661006, 2190: 2202.3666871716778}" ;
        :l2_filename = "/work/ESTUARY/sentinel/data_processed_polymer/run_4594619/data/S2A_MSIL1C_20170903T160131_N0205_R054_T18SVJ_20170903T160146_10m.nc" ;
        :l2_format = "NETCDF4_CLASSIC" ;
data:
}

**Comment by snowman2** [comment]
This may be helpful: https://corteva.github.io/rioxarray/stable/getting_started/crs_management.html
